app/views/idv/session_errors/throttled.html.erb

```diff
@@ -20,11 +20,7 @@
       <p>
         <%= t(
               'errors.doc_auth.throttled_text_html',
-              timeout: distance_of_time_in_words(
```

Comment:
What do you think of dropping the added module? I'm not sure if the extra layer is worth the indirection.

Reply:
I'll remove it, but it made the ruby easier to debug, which is valuable in itself IMO.